packages/venia-concept/src/RootComponents/Category/categoryContent.js

```diff
@@ -9,16 +9,18 @@ class CategoryContent extends Component {
         const { classes, pageControl, data, pageSize } = this.props;
         const items = data ? data.category.products.items : null;
         const title = data ? data.category.description : null;
+        const categoryTitle = data ? data.category.name : null;
```

Comment:
@jimbo,  @supernova-at, maybe it makes sense to add `lodash.get` package to the project to avoid such ternary expression, what do you think?